roadmap/implementers-guide/src/node/backing/prospective-parachains.md

```diff
@@ -0,0 +1,163 @@
+# Prospective Parachains
+
+## Overview
+
+**Purpose:** Tracks and handles prospective parachain fragments and informs
+other backing-stage subsystems of work to be done.
+
+"prospective":
+- [*prə'spɛktɪv*] adj.
+- future, likely, potential
+
+Asynchronous backing changes the runtime to accept parachain candidates from a
+certain allowed range of historic relay-parents. This means we can now build
+*prospective parachains* – that is, trees of potential (but likely) future
+parachain blocks. This is the subsystem responsible for doing so.
+
+Other subsystems such as Backing rely on Prospective Parachains, e.g. for
+determining if a candidate can be seconded. This subsystem is the main
+coordinator of work within the node for the collation and backing phases of
+parachain consensus.
+
+Prospective Parachains is primarily an implementation of fragment trees. It also
+handles concerns such as:
+
+- the relay-chain being forkful
+- session changes
+- predicting validator group assignments
```

Comment:
Nit: @slumber Indicated in a separate comment this isn't actually happening. Just wanted to tag this as another place to potentially remove mention.